task_sdk/tests/execution_time/test_task_runner.py

```diff
@@ -248,43 +248,6 @@ def test_run_basic_skipped(time_machine, mocked_parse, make_ti_context):
         )
 
 
-def test_startup_basic_templated_dag(mocked_parse, make_ti_context):
```

Comment:
Why remove this?

Reply:
I see you are handling both in a new test

Reply:
This test is more like a subset of the `test_startup_and_run_dag_with_templated_fields` which is a more generic one with different parameters. This was a specific case of BashOperator,